### PR TITLE
fix: events board teleportation

### LINKED ISSUE #1268: goerli: Events-API-Board | missing 'jump in' modal
### **Explorer Alpha build version:**

- **0.3.477-g5ad20358-main**
- **v0.11.0**
- **v0.12.0**

### **Issue Description:**
At (78,1) when clicking the banner, the modal to 'jump in' doesn't appear.

### **Steps to reproduce:**

1. /goto goerli
2. go to escene (78,1)
3. Click on any board

### **Expected behavior:**

https://github.com/decentraland/unity-explorer/assets/146741383/f93d3bcf-d747-49c2-8bb4-a28b787b3980

### **Current behavior:**

https://github.com/decentraland/unity-explorer/assets/146741383/49ea0e70-0031-4761-a13a-de5408393007

#### **Additional Notes:**
Windows 11. 12th Gen Intel(R) Core(TM) i7-12650H 2.30 GHz.
nvidia geforce RTX

### PR DESCRIPTION
**Pull request by lorux0**
## What does this PR change?

Fixes #1268 

The fix consists to ensure that the teleport coordinates are sent as numbers from js->c#. The events board api scene was passing the parameters as strings.

Also the scene was stopping its execution after some time because it internally has javascript runtime errors, which also happens in unity-renderer, but they dont affect the user experience:
```
ScriptEngineException: TypeError: Cannot read properties of undefined (reading 'image')
    at displayEvent (Script [19]:33185:43)
    at Object.switchEventSystem [as fn] (Script [19]:33170:11)
    at Object.update (Script [19]:31680:26)
    at async Object.onUpdate (Script [19]:34927:3)
    at async AsyncFunction.__internalOnUpdate (Script [21]:13:21)
```

The tolerance of javascript errors has been increased. However is there is any other scene that exceeds this limit, the creator will have to fix the errors in javascript level.

Finally, there has been an intensive debugging session in the simple fetch api (since this scene uses it), so i needed to make it more descriptive. The core functionality should remain the same.

## How to test the changes?

1. Follow issue steps
2. The teleport popup should be displayed as expected with the given coordinates

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by anicalbano — COMMENTED** [review]
🔴 It's still not possible to click on the boards


https://github.com/user-attachments/assets/45fe2455-fa10-4fc8-b8ea-87bd4cbfa127

**Review by anicalbano — DISMISSED** [review]
🟢 Reviewed by QA

https://github.com/user-attachments/assets/88bbdab6-047e-4ebc-8af5-b98bf36b4e3e

File: Explorer/Assets/Scripts/SceneRunner/Scene/ExceptionsHandling/SceneExceptionsHandler.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ namespace SceneRunner.Scene.ExceptionsHandling
     public class SceneExceptionsHandler : ISceneExceptionsHandler
     {
         // Experiment with this, maybe tolerance should be 0
-        internal const int ECS_EXCEPTIONS_PER_MINUTE_TOLERANCE = 3;
+        internal const int ECS_EXCEPTIONS_PER_MINUTE_TOLERANCE = 30;
```
Comment (mikhail-dcl) [inline Explorer/Assets/Scripts/SceneRunner/Scene/ExceptionsHandling/SceneExceptionsHandler.cs:17]: Please separate ECS and JS tolerance, don't increase the former

File: Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs [hunk 2]
```diff
@@ -22,21 +22,21 @@ public UnityOpsApi(V8ScriptEngine engine, ISceneModuleHub moduleHub, V8Script sc
         }
 
         [UsedImplicitly]
-        public void Log(object message)
+        public void Log(string message)
```
Comment (mikhail-dcl) [inline Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs:25]: Does it fix the problem with printing `object` instead of the content?

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]

File: Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs (hunk 2 above)
Comment (lorux0) [reply Explorer/Assets/Scripts/SceneRuntime/Apis/UnityOpsApi.cs:25]: Nope, its the same. However i've checked that we always receive a `string` in this methods as it is implemented here: https://github.com/decentraland/unity-explorer/blob/97b2196efdfe39ce2c86ba28b1420c0d16d24793/Explorer/Assets/StreamingAssets/Js/Init.js#L39
So i dont see a reason on why keeping the object there

**Review by mikhail-dcl — APPROVED** [review]

**Pull request merged**